consensus/hotstuff/committees/consensus_committee.go

```diff
@@ -11,35 +11,83 @@ import (
 	"github.com/onflow/flow-go/model/flow"
 	"github.com/onflow/flow-go/model/flow/filter"
 	"github.com/onflow/flow-go/state/protocol"
-	"github.com/onflow/flow-go/state/protocol/seed"
 )
 
-var errSelectionNotComputed = fmt.Errorf("leader selection for epoch not yet computed")
+// ErrViewForUnknownEpoch is returned when a by-view query is made with a view
+// outside all cached epochs. This can happen when a query is made for a view in the
```

Comment:
wording suggestion that is easier to understand for my non-native-english-speaking brain 🤷  

```suggestion
// ErrViewForUnknownEpoch is returned when Epoch information is queried for a view that is 
// outside of all cached epochs. This can happen when a query is made for a view in the
```